src/99-appendices/09-entities.md

```diff
@@ -151,6 +154,26 @@ sampling frequencies).
 In such case use different labels.
 For example: `_recording-contrast`, `_recording-saturation`.
 
+## part
+
+Full name: Part
+
+Format: `part-<label>`
+
+Definition: This entity is used to indicate which component of the complex
+representation of the MRI signal is represented in voxel data.
+The `part-<label>` key/value pair is associated with the DICOM tag
+`0008,9208`.
+Allowed label values for this entity are `phase`, `mag`, `real` and `imag`,
+which are typically used in `mag/phase` or `real/imag` pairs.
```

Comment:
see comment above on that I find the `mag/phase` suggestive of being some kind of syntax we support (but we don't)